docs/key-issues/gpus.md

```diff
@@ -180,3 +204,84 @@ Potential solutions on the PyPI side include:
+and
+[here](https://docs.nvidia.com/cuda/cuda-c-best-practices-guide/index.html#distributing-the-cuda-runtime-and-libraries)
+for more information).
+
+CUDA drivers have always been backwards compatible. Any code that runs when
+some driver version X is installed will always work correctly when run with
+some newer driver version Y>X. As briefly discussed above, as of CUDA 11.0
+CUDA also promises [minor version
+compatibility](https://docs.nvidia.com/deploy/cuda-compatibility/index.html#minor-version-compatibility)
+(MVC). This compatibility guarantees that CUDA code compiled using a certain
+version of the CUDA toolkit will be compatible with any driver version within
+the same major release. This behavior is useful because it is often easier for
+users to upgrade their CUDA runtime than it is to upgrade the driver,
+especially on shared machines. For instance, the CUDA toolkit may be installed
+using conda, while the driver library cannot be. An example of leveraging MVC
+would be compiling code against the CUDA 11.5 runtime library and then running
+it on a system with a CUDA 11.2 driver installed. However there are some
+caveats with MVC:
+
+- If the code uses any features that were introduced in a later driver version
+  than the installed version, it will still fail to run. However, it will be a
+  runtime failure in the form of a `cudaErrorCallRequiresNewerDriver` CUDA
+  error, rather than a linker error or some similarly opaque issue. One
+  solution to this problem is for libraries to use runtime checks of the CUDA
+  version (using e.g. `cudaDriverGetVersion`) to only use supported features on
+  the installed driver.
+- NVRTC did not start supporting MVC until CUDA 11.3. Therefore, code that uses
+  NVRTC for JIT compilation must have been compiled with a CUDA version >= 11.3.
+- MVC only applies to CUDA code, not
+  [PTX](https://docs.nvidia.com/cuda/parallel-thread-execution/). PTX is an
+  instruction set that the CUDA driver library can JIT-compile to machine
+  instructions. The standard [CUDA compilation
+  pipeline](https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#the-cuda-compilation-trajectory)
+  includes the translation of CUDA code into PTX, but in addition developers
+  may write PTX code directly rather than CUDA for performance since it offers
+  avenues for optimization that are either unavailable or very difficult to
+  access with CUDA C(++). However, since PTX code does not support MVC, PTX
+  compiled with a particular CUDA runtime may not work if run on a system with
+  an older driver. This fact has two consequences. First, libraries that
+  package PTX code will not benefit from MVC. Second, libraries that leverage
+  any sort of JIT-compilation pipeline that generates PTX code will _also_ not
+  support MVC. The latter can lead to more surprising behaviors, such as if a
+  user has a newer CUDA runtime than driver and then uses
```

Comment:
Another use of "runtime", where I think the toolkit version is really what's important:

```suggestion
  user has a newer CUDA toolkit than driver and then uses
```